### DISCUSSION TITLE
stuck at Repair setup and hold violations

### DISCUSSION THREAD

**Discussion by velmuruganvmtor** [The-OpenROAD-Project/OpenROAD-flow-scripts#1876] [Q&A]
i am stuck or froze at Repair setup and hold violations 
in terminal :
Repair setup and hold violations...
TNS end percent 100
[INFO RSZ-0094] Found 601 endpoints with setup violations.
please help.

**Comment by maliberty** [comment]
Try adding -verbose to repair_timing to get more info

**Reply by velmuruganvmtor** [reply]
where to apply the command and give a example please

**Reply by maliberty** [reply]
https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/4f99f1dda7d0171448a2e1a0facbc2e22e633099/flow/scripts/cts.tcl#L86

**Reply by velmuruganvmtor** [reply]
like repair_timing {*}$verbose or repair_timing {verbose}$additional_args

**Reply by velmuruganvmtor** [reply]
also i stuck at Found 1 endpoints with hold violations

**Reply by maliberty** [reply]
repair_timing -verbose {*}$additional_args 

It would be easier to help with a test case

**Reply by velmuruganvmtor** [reply]
any solution for this 
also i stuck at Found 1 endpoints with hold violations

**Reply by velmuruganvmtor** [reply]
i added that code to that line but no changes in terminal

**Reply by maliberty** [reply]
Then you need to open an issue with a test case
